javaagent-bootstrap/src/test/java/io/opentelemetry/javaagent/bootstrap/PatchLoggerTest.java

```diff
@@ -804,18 +808,19 @@ public boolean equals(@Nullable Object obj) {
       MethodSignature other = (MethodSignature) obj;
       return Objects.equals(name, other.name)
           && Objects.equals(parameterTypes, other.parameterTypes)
-          && Objects.equals(returnType, other.returnType);
+          && Objects.equals(returnType, other.returnType)
+          && modifiers == other.modifiers;
```

Comment:
might be better to use `Modifier.isStatic(modifiers) == Modifier.isStatic(other.modifiers)`